specs/protocol/deputy-pause-module.md

```diff
@@ -0,0 +1,387 @@
+expiration timestamps.
+
+#### Impact
+
+**Severity: Medium**
+
+If this invariant is broken, authorizations created by the Deputy could be used long after the
+authorization was created (perhaps indefinitely). Such signatures would have the same power as
+access to the key itself. This increases the risk that the pause is triggered outside of the
+standard process by which such a pause is approved.
+
+#### Mitigations
+
+- Enforce that all signatures contain an expiration timestamp.
+- Enforce that the expiration timestamp can only be so far into the future.
+
+#### Dependencies
+
+- [aDPM-002](#adpm-002-deputy-key-is-not-compromised)
+- [aDPM-004](#adpm-004-deputy-key-will-not-sign-excessively-long-expirations)
+- [aDPM-007](#adpm-007-openzeppelin-ecdsa-and-eip712-contracts-are-free-of-critical-bugs)
+
+### iDPM-005: Deputy authorizations must not be replayable
+
+A Deputy authorization must apply to a specific `DeputyPauseModule` on a specific blockchain as
+identified by its chain ID. An authorization created for one `DeputyPauseModule` must not be
+reusable on any other `DeputyPauseModule`. An authorization must be usable once and the same
+authorization must not be usable again in any `DeputyPauseModule`.
+
+#### Impact
+
+**Severity: High**
+
+If this invariant is broken, a Deputy authorization created by the same Deputy address for
+another `DeputyPauseModule` or a previous authorization for the same module could be reused and
+would result in the Superchain-wide pause being triggered outside of the standard process by which
+such a pause is approved.
+
+#### Mitigations
+
+- EIP-712 signature verification including a specific contract address and chain ID.
+- Enforce unique nonces on signatures to prevent signature reuse within the same contract.
+- Utilize unique Deputy accounts for each module instance or network.
+
+#### Dependencies
+
+- [aDPM-007](#adpm-007-openzeppelin-ecdsa-and-eip712-contracts-are-free-of-critical-bugs)
+
+## Implementation Spec
+
+### constructor
+
+- Sets the EIP-712 domain name to "DeputyPauseModule".
+- Sets the EIP-712 domain version to "1".
+- Takes the address of the Operations Safe as an authorized input.
+- Takes the address of the `DeputyGuardianModule` as an authorized input.
+- Takes the address of the `SuperchainConfig` as an authorized input.
+- Takes a max signature validity time as an authorized input.
+- Takes the address of the Deputy as an authorized input.
+- Takes a signature from the Deputy over a known EIP-712 message.
+- Must verify that the signature was produced by the provided Deputy address.
+- Must verify that the max signature validity time is greater than 1 day and less than 365 days.
+- Must verify that the Operations Safe exposes some known Safe getter function.
+- Must verify that the `DeputyGuardianModule` exposes some known getter function.
+- Must verify that the `SuperchainConfig` exposes some known getter function.
```

Comment:
I'd love to see this info presented as natspec on an interface, I think that makes it a bit more intuitive to process, especially given that 6 lines are dedicated to input args.